client/src/components/SavedSongs/SavedSong.js

```diff
@@ -34,7 +34,7 @@ export default function SavedSong({
         <ListItem>
           <ButtonWrapper>
             <ButtonPrepareToDelete
```

Comment:
Maybe you could check, if there is a way to get rid of the ``ButtonWrapper``, as it wrapps only one component. Perhaps the placement via grid could be done in a different way?